### PR TITLE
Adding two randomHash options

### LINKED ISSUE #24: Open to adding random #hashes
I use a lot of hashes like #iW8saj in code to be able to easily be able to grep from a log message to a place in the code that generated the message. We use them liberally, generate them randomly on demand, and try to surface them to users. A screenshot including one of these tells us exactly which error message the user is seeing.

Are you open to a PR adding this option?

### PR DESCRIPTION
**Pull request by chmac**
Fixes #24 

Added this to use it locally. Figured I'd send a PR. Not sure if I've caught all the correct places to add the command, but it does seem to work on my local setup.

It's also a bit wacky to add 2 commands, but I always use a fixed length hash, so having to type the number 6 every time I want a 6 digit hash was a non-starter for me personally. Can understand if you think it only makes sense to support one of them. I also added a placeholder which is not actually used because the command fails if the entered value is not a number. Hmm, maybe that wants fixing...

Changes to make:

* [ ] Add unit tests
* [ ] Remove fixed length version
* [ ] Solve default value bug
* [ ] Update readme

### CODE REVIEW

File: src/random.js [hunk 1]
```diff
@@ -43,6 +44,14 @@ export const randomString = ({chance = chanceInstance, inputValue = VALUE_DEFAUL
   return chance.word({length: inputValue})
 }
 
+export const randomHash = ({chance = chanceInstance, inputValue = VALUE_DEFAULT_HASH_LENGTH}) => {
```
Comment (jrebocho) [inline src/random.js]: What do you think of having some unit tests for this function?

**Comment by jrebocho** [comment]
Can you also update the README with the new command?

File: src/random.js (hunk 1 above)
Comment (chmac) [reply src/random.js]: 👍 

I implemented it quickly so I could get it running locally, and wasn't sure I had added it in all the correct places. Will add tests.

**Comment by chmac** [comment]
@jrebocho Yes, will update the README.

One part I didn't understand fully was how to set a default value. I copied an example of a placeholder value, but that doesn't work the same. It's necessary to enter a value into the box, otherwise I see a "you did not enter a number" error. Is there an example of a method which has a default value that I could copy? Or any advice on how to have a validation rule that says "either no value, or a number" rather than just "must be a number"?

**Comment by jrebocho** [comment]
Hi @chmac, I will need to tweak a bit the code that adds the commands to support default values. You can proceed with this without the default value and I can add it later before publishing the extension.

**Comment by chmac** [comment]
Just a quick follow up to say I never got around to adding tests to my PR so will close this PR and issue.

I ended up creating an [extension just for this](https://marketplace.visualstudio.com/items?itemName=chmac.insert-random-hashtag). I see that the version with the option of a string length makes more sense in your plugin, and the other option is the one I use daily, so figured I'd dabble in publishing a plugin.

Thanks, your plugin was helpful in figuring out how to do all this stuff. 👍

**Pull request closed without merging**